cetmix_tower_git/models/cx_tower_server.py

```diff
@@ -36,3 +36,34 @@ def _get_fields_for_yaml(self):
             "git_project_rel_ids",
         ]
         return res
+
+    def _command_runner_file_using_template_create_file(
+        self,
+        file_template_id,
+        server_dir,
+        **kwargs,
+    ):
+        """Override to create git project relation
+        when creating a file using a template.
+        """
+        file_id = super()._command_runner_file_using_template_create_file(
```

Comment:
the `create_file` function of the `cx.tower.file.template` model returns a [file](https://github.com/cetmix/cetmix-tower/blob/ac9775b6a0723d3f5575c036ab6f165ba6429755/cetmix_tower_server/models/cx_tower_file_template.py#L146), not a file ID.